packages/Notification/Notification.md

```diff
@@ -17,6 +23,8 @@ By default, notifications will be displayed in the `instructional` variant.
   not use other complex components or paragraphs.
 - Use complete sentences, while keeping the content succinct.
 - Dismissible notifications should only be used for branded or general style notifications where there is no action needed from the user.
+- If the `variant` is `success` or `error`, there should be a `Box` with `between` level 3 between the icon and the text
```

Comment:
I don't think this line should belong on our docs

Reply:
This was part of the misunderstanding of this ticket, I'll remove it.